packages/build-tools/orchestrator/src/tmux/README.md

```diff
@@ -0,0 +1,18 @@
+# TMux @ Orchestrator
+
+This is a Tmux wrapper for Deno.
+
+* By default it loads `tmux.launch.json` to setup the initial panes and programs. 
+
+Format:
+```json
+{
+    name: string,
+    command: string,
+    split_horizontal?: boolean,
+    split_vertical?: boolean,
+}[]
+```
+If neither split_vertical or split_horizontal is defined, then the command is executed in the first pane.
+
+* It uses the `tmux.confg` as configuracion file.
```

Comment:
Probably we should start populating the `docs` folder with stuff like this to document how Paima Engine v2 works

Reply:
I created a issue to track/do this.
should we try to keep docusaurus? 
I gave it a quick try with deno, but didn't work for me. 
We also have some alternatives that as https://github.com/getzola/zola (it's distributed as single binary, and there is npm:zola-bin )